public/assets/css/course_tab.css

```diff
@@ -0,0 +1,95 @@
+.course-tab-section {
+    --padding: 2em;
+    display: flex;
+    justify-content: flex-end;
+    padding: var(--padding); 
+    background-color: var(--grey);
+    flex-direction: column;
+    flex-wrap: wrap;
+    font-family: var(--text-font);
+}
+
+.tab {
+    display: flex;
+    flex-direction: column; 
+    align-items: center; 
+}
+
+.tab-title {
+    --margin-top: 0.8em;
+    --font-size: 1.2em;
+    margin-top: var(--margin-top);
+    font-size: var(--font-size);
+    font-weight: bold;
+    text-align: center;
+}
+
+.tab-text {
+    display: flex;
+    flex-wrap: wrap;
+}
+
+.btn-insc {
+    display: flex;
+    justify-content: center;
+}
+
+.btn-inscription {
+    --margin: 3em;
+    --width: 10em;
+    margin-bottom: var(--margin);
+    margin-top: var(--margin);
+    border-radius: 10em;
+    background-color: var(--secondary-color);
+    height: var(--margin);
+    width: var(--width);
+    color: var(--dark);
+    border: none;
+}
+
+.btn-course {
+    --margin: 0.5em;
+    --height: 3em;
+    --width: 13em;
+    --border-radius: 2em;
```

Comment:
I still feel like you have a lot of different values for padding, margins, etc... maybe you should change that in the future and use multiples of one single value (4 and 8px are commonly used for paddings and margins)